README.md

```diff
@@ -3718,7 +3725,10 @@ resource cleanup when possible.
   ```
 
 * <a name="consistent-string-literals"></a>
-  Prefer single quotes for string literals.
+  Prefer double quotes when
+    * writing natural language (since it might contain apostrophes)
```

Comment:
I think this is really just "prefer single quotes for string literals; use double quotes for interpolation or apostrophes".